### DISCUSSION TITLE
Can we use it without agora credentials?

### DISCUSSION THREAD

**Discussion by hemantrao12** [TEN-framework/ten-framework#537] [General]
Can we use it without agora credentials?

**Comment by plutoless** [comment]
an important point for using ten agent project is to use rtc to improve multimodal data transport. the default option we provided is agora, as this is our only supported option and we believe rtc is mandatory in this case, so currently yes you will have to use it with agora credentials (or environment check will fail), the registrations are pretty easy on https://agora.io
however it's not a hard restriction for ten_framework to run with agora credentials, you can use ten_framework to orchestrate what ever you like.

**Comment by remisharrock** [comment]
related to https://github.com/TEN-framework/TEN-Agent/discussions/288
